test/goaway_test.go

```diff
@@ -57,21 +57,20 @@ func (s) TestGracefulClientOnGoAway(t *testing.T) {
 	const maxConnAge = 100 * time.Millisecond
 	const testTime = maxConnAge * 10
 
+	lis, err := net.Listen("tcp", "localhost:0")
+	if err != nil {
+		t.Fatalf("Failed to create listener: %v", err)
+	}
+
 	ss := &stubserver.StubServer{
```

Comment:
nit: this can be ss1 and then others can ss2, ss3 and so on.

Reply:
Done